docs/apache-airflow/howto/operator/bash.rst

```diff
@@ -184,6 +184,48 @@ exit code if you pass ``skip_on_exit_code``).
             :end-before: [END howto_operator_bash_skip]
 
 
+Result processor
```

Comment:
Don't forget updating the docs :)

Reply:
Yeah the doc in example should be corrected :)

Reply:
Just did it